frontend/mulighetsrommet-cms/schemas/tiltaksgjennomforing.js

```diff
@@ -0,0 +1,225 @@
+import { GrDocumentPerformance } from "react-icons/gr";
+
+export default {
+  name: "tiltaksgjennomforing",
+  title: "Tiltaksgjennomføring",
+  type: "document",
+  icon: GrDocumentPerformance,
+  fields: [
+    {
+      name: "tiltakstype",
+      title: "Tiltakstype",
+      type: "reference",
+      to: [{ type: "tiltakstype" }],
+      validation: (Rule) => Rule.required(),
+    },
+    {
+      name: "title",
+      title: "Navn på tiltaksgjennomføring",
+      type: "string",
+      validation: (Rule) => Rule.required(),
+    },
+    {
+      name: "tiltaksnummer",
+      title: "Tiltaksnummer",
+      type: "number",
+    },
+    {
+      name: "leverandor",
+      title: "Leverandør",
+      type: "string",
+    },
+    {
+      name: "lokasjon",
+      title: "Lokasjon",
+      type: "string",
+    },
+    {
+      name: "enheter",
+      title: "Enheter",
+      description:
+        "Hvilke enheter skal ha tilgang til denne tiltaksgjennomføringen?",
+      type: "document",
+      fields: [
+        {
+          name: "fylke",
+          title: "Fylke",
+          type: "string",
+          options: {
+            layout: "dropdown",
+            list: [
+              { title: "Innlandet", value: "innlandet" },
+              { title: "Trøndelag", value: "trondelag" },
+              { title: "Vest-Viken", value: "vestviken" },
+              { title: "Øst-Viken", value: "ostviken" },
+            ],
+          },
+        },
+        //innlandet
+        {
+          name: "ringsaker",
+          title: "Ringsaker",
+          type: "boolean",
+          initialValue: false,
+          hidden: ({ parent }) =>
+            parent?.fylke === "trondelag" ||
+            parent?.fylke === "vestviken" ||
+            parent?.fylke === "ostviken" ||
+            parent?.fylke === undefined,
```

Comment:
Er det ikke bedre å gjøre en omvendt sjekk? Altså sjekk at `parent?.fylke !== "innlandet"`? Da vil man slippe å tenke på å oppdatere alle `||`-sjekkene når vi legger til flere fylker senere. Denne kommentaren gjelder altså for alle sjekkene nedover.